tests/functional_tests/ffront_tests/test_lowering.py

```diff
@@ -51,17 +49,21 @@ def copy_field(inp: Field[..., "float64"]):
     assert lowered.expr == im.deref_("inp")
 
 
-@pytest.mark.skip(reason="Constant-to-field promotion not enabled yet.")
-@pytest.mark.skip(reason="Iterator IR does not allow arg-less lambdas yet.")
-def test_constant():
-    def constant():
-        return 5
+def test_scalar_arg():
+    def constant_arg(bar: Field[..., int64], alpha: int64) -> Field[..., int64]:
+        return alpha * bar
 
     # ast_passes
-    parsed = FieldOperatorParser.apply_to_function(constant)
+    parsed = FieldOperatorParser.apply_to_function(constant_arg)
     lowered = FieldOperatorLowering.apply(parsed)
 
-    assert lowered.expr == im.deref_(im.lift_(im.lambda__()(5)))
+    reference = im.deref_(
```

Comment:
How can you deref `alpha`. Is this WIP and will be lifted in the future? I would assume that this should work the same way as literals below, i.e. no deref.

Reply:
This is strictly speaking part of the next feature, scalar args support.
That will introduce automatic Field / Iterator creation for scalar arguments, so scalar arguments can always be treated as `Field[[], dtype]` inside the field operator / fundef (not decided on which level yet).